driver/makedepend.ml

```diff
@@ -591,6 +593,8 @@ let main () =
         " Dump the delayed dependency map for each map file";
      "-I", Arg.String (add_to_list Clflags.include_dirs),
         "<dir>  Add <dir> to the list of include directories";
+     "-nocwd", Arg.Set nocwd,
+        " Do not include current working dir in list of include directories";
```

Comment:
Sorry, I didn't reread this text attentively enough: I am not fond of the `include` repetition or the `dir` abbreviation. Overall, I like @dbuenzli's proposition better:
>  Do not add current working directory to the list of include directories

Apart from this, the PR looks to merge.

Reply:
Updated